### PR TITLE
Autofix SA1518: No extra blank lines at the end of the file

### PR DESCRIPTION
**Pull request by xtqqczze**
# PR Summary

Autofix SA1518: The code must not contain blank lines at the end of the file.

https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1518.md

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [NA] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [NA] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [NA] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [NA] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [NA] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [NA] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

---
This change is [Reviewable](https://reviewable.io/reviews/powershell/powershell/13574)

### CODE REVIEW

**Review by iSazonov — APPROVED** [review]

**Comment by daxian-dbw** [comment]
@PowerShell/powershell-maintainers reviewed this PR and agreed to reject it. We have been telling contributors to add a new line at the end for quite some time, because GitHub code difference usually complains about not having a new line at the end.

**Comment by xtqqczze** [comment]
@daxian-dbw According to the [documentation for SA1518](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1518.md), the default setting for this rule is: "Files are allowed to end with a single newline character, but it is not required".

To be clear, while this PR removes the necessary blank lines at the the of files, it does not remove the single newline charactor at the end of the last line (which may in fact cause issues with diff tools).

**Comment by iSazonov** [comment]
@daxian-dbw The PR remove **_extra second empty_** line at EOF.

File: src/Microsoft.PowerShell.Commands.Management/commands/management/AddContentCommand.cs [hunk 1]
```diff
@@ -84,4 +84,3 @@ internal override bool CallShouldProcess(string path)
         #endregion protected members
```
Comment (Steg17) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/AddContentCommand.cs:84]: #and region

Comment (Obmorok070) [reply src/Microsoft.PowerShell.Commands.Management/commands/management/AddContentCommand.cs:84]: G

**Comment by xtqqczze** [comment]
@iSazonov Please reopen.

**Comment by xtqqczze** [comment]
> 
> 
> @daxian-dbw The PR remove **_extra second empty_** line at EOF.

@daxian-dbw Can you please re-review, I would like this merged early for `7.2.0-preview.1`.

**Review by TravisEz13 — COMMENTED** [review]
Reviewed 57 files

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 7 days

**Comment by iSazonov** [comment]
@TravisEz13 Please continue your review.

**Review by iSazonov — APPROVED** [review]

**Pull request merged**